### PR TITLE
Fixed ACLs for openshift console in end-to-end example

### PR DESCRIPTION
**Pull request by vbontempi**
### Description

Added ACLs to allow openshift console pods to reach oAuth server in order to start successfully and added flag to disable these ACLs, including ingress healthcheck operator ones, in end-to-end example

### Release required?

- [x] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

##### Release notes content

Added ACLs to allow openshift console pods to reach oAuth server in order to start successfully in end-to-end example
Added flag to disable these ACLs, including ingress healthcheck operator ones in end-to-end example

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [X] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

File: examples/end-to-end-example/variables.tf [hunk 1]
```diff
@@ -69,3 +69,9 @@ variable "disable_public_endpoint" {
   description = "Whether access to the public service endpoint is disabled when the cluster is created. Does not affect existing clusters. To change a public endpoint to private, create another cluster with the variable set to true or see [Switching to the private endpoint](https://cloud.ibm.com/docs/containers?topic=containers-cs_network_cluster#migrate-to-private-se)."
   default     = false
 }
+
+variable "allow_ingress_console_traffic" {
```
Comment (ocofaigh) [inline examples/end-to-end-example/variables.tf:73]: I wouldn't bother exposing a variable for this - just do it in the example

**Comment by vbontempi** [comment]
Skipped upgrade test as expected changes happening
```
Address: module.vpc.ibm_is_network_acl.network_acl["ocp-all-upg-p1fkeh-acls"]
        	            	Actions: [update]
        	            	DIFF:
        	            	  Before: 
        	            		{"rules":"SECURE_VALUE_HIDDEN_HASH:-2990e18eebf63606104e98ad461c909425b6c50eddfa4645bbb9c3d4"}
        	            	  After: 
        	            		{"rules":"SECURE_VALUE_HIDDEN_HASH:-41a87c5f26b878d71081ff792a20888e78d5e7586f4046f29a3ae357"}
```

**Review by ocofaigh — APPROVED** [review]

**Pull request merged**